industrial_ci/src/util.sh

```diff
@@ -254,6 +254,7 @@ function ici_retry {
 function ici_quiet {
     local out; out=$(mktemp)
     # shellcheck disable=SC2216
+    # shellcheck disable=SC2260
```

Comment:
@ipa-mdl There is a new shellcheck error popping up here in the Github Action check

Reply:
I cherry-picked your fix to #506

Reply:
Thanks!